scss/message/_message.scss

```diff
@@ -142,10 +142,10 @@
     }
 
     .module-message__img-attachment {
```

Comment:
some wide pictures get cut off, maybe make the height smaller in this case if possible.

Reply:
We can't dynamically set the height of images because this would make the scrollbar jump.

Reply:
I thought the saving image dimensions in the db api was already done, but now I see that its only partly implemented